cumulusci/tasks/bulkdata/generate_from_yaml.py

```diff
@@ -32,7 +36,24 @@ class GenerateDataFromYaml(BaseGenerateDataTask):
             "description": "A path to put a mapping file inferred from the generator_yaml",
             "required": False,
         },
+        "num_records_tablename": {
+            "description": "A string representing which table to count records in."
+        },
+        "continuation_file": {
```

Comment:
The descriptions for these two options might not provide enough guidance on how to use them.

Reply:
num_records_tablename and continuation_file or

continuation_file and generate_continuation_file?

Reply:
The latter two.

Reply:
Is it clearer in 15e5f3ad ?

Reply:
Very much so - thank you!